RecastDemo/Source/main.cpp

```diff
@@ -817,33 +819,30 @@ int main(int /*argc*/, char** /*argv*/)
 					{
 						ctx.dumpLog("Build log %s:", meshName.c_str());
 					}
-					
-					if (geom || sample)
+
+					const float* bmin = 0;
+					const float* bmax = 0;
+					if (geom)
 					{
-						const float* bmin = 0;
-						const float* bmax = 0;
-						if (geom)
-						{
-							bmin = geom->getNavMeshBoundsMin();
-							bmax = geom->getNavMeshBoundsMax();
-						}
-						// Reset camera and fog to match the mesh bounds.
-						if (bmin && bmax)
-						{
-							camr = sqrtf(rcSqr(bmax[0] - bmin[0]) +
-										 rcSqr(bmax[1] - bmin[1]) +
-										 rcSqr(bmax[2] - bmin[2])) / 2;
-							cameraPos[0] = (bmax[0] + bmin[0]) / 2 + camr;
-							cameraPos[1] = (bmax[1] + bmin[1]) / 2 + camr;
-							cameraPos[2] = (bmax[2] + bmin[2]) / 2 + camr;
-							camr *= 3;
-						}
-						cameraEulers[0] = 45;
-						cameraEulers[1] = -45;
-						glFogf(GL_FOG_START, camr * 0.2f);
-						glFogf(GL_FOG_END, camr * 1.25f);
+						bmin = geom->getNavMeshBoundsMin();
+						bmax = geom->getNavMeshBoundsMax();
 					}
-					
+					// Reset camera and fog to match the mesh bounds.
+					if (bmin && bmax)
+					{
+						camr = sqrtf(rcSqr(bmax[0] - bmin[0]) +
+										rcSqr(bmax[1] - bmin[1]) +
+										rcSqr(bmax[2] - bmin[2])) / 2;
+						cameraPos[0] = (bmax[0] + bmin[0]) / 2 + camr;
+						cameraPos[1] = (bmax[1] + bmin[1]) / 2 + camr;
+						cameraPos[2] = (bmax[2] + bmin[2]) / 2 + camr;
+						camr *= 3;
+					}
+					cameraEulers[0] = 45;
+					cameraEulers[1] = -45;
+					glFogf(GL_FOG_START, camr * 0.2f);
+					glFogf(GL_FOG_END, camr * 1.25f);
+
```

Comment:
This block is copy/pasted in 2 other places.  Should those be updated as well?  Lines 636 and 708